Format dates like "Mon Jan 17th"

Benjamin-GitHub opened the pull request:
Date format is set now both in WeeklyTable & BookingScreen as requested like "Mon Jan 17th" with using of "Moment JS" library.

Benjamin-GitHub committed:
Format dates like "Mon Jan 17th"

gregdyke commented on client/src/pages/WeeklyTable.js:
```diff
@@ -34,9 +35,9 @@ const WeeklyTable = ({ bookings, rowsCount }) => {
+								{moment(date).format("ddd MMM Do")}{" "}
 								<ModalBookingScreen
-									bookingDate={new Date(date).toLocaleDateString()}
+									bookingDate={moment(date).format("ddd MMM Do")}
```
I'm almost certain this prevents the BookingScreen from being able to save bookings. So I'm surprised you don't see the same problem?

gregdyke reviewed (changes requested):
looks great, except I think this is related to why BookingScreen doesn't create bookings any more

Benjamin-GitHub committed:
Create formatBookingDate in util.js

Use formatBookingDate in ModalBookingScreen and WeeklyTable
Fix date formatting bug that prevented POST /bookings from working

gregdyke reviewed (approved):
I think in this case, using moment is fine
